pkgpanda/docker.windows/dcos-builder/Dockerfile

```diff
@@ -33,13 +34,19 @@ COPY .\setup-golang.ps1 \temp\
 RUN  powershell.exe -file c:\temp\setup-golang.ps1
 ENV GOPATH 'c:\gopath'
 
+ADD https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-win.zip c:\temp\ninja.zip
+RUN powershell -command "& new-item -itemtype directory ${env:ProgramFiles(x86)}\ninja"
+RUN powershell -command "& Expand-Archive c:\temp\ninja.zip ${env:ProgramFiles(x86)}\ninja"

```

Comment:
I'm not sure. It may have been added to support building a package that's not part of this PR. I'll see if I can remove this without breaking anything.